src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -22,6 +22,19 @@ ucs_stats_class_t uct_rc_mlx5_iface_stats_class = {
 };
 #endif
 
+
+ucs_config_field_t uct_common_mlx5_config_table[] = {
+#if HAVE_IBV_EXP_DM
+    {"DM_SEG_SIZE", "1k",
+     "Device Memory segment size (0 - disabled)",
+     ucs_offsetof(uct_common_mlx5_iface_config_t, dm.seg_len), UCS_CONFIG_TYPE_MEMUNITS},
+    {"DM_SEG_NUM", "1",
```

Comment:
i'd suggest to rename to "DM_SIZE" and "DM_COUNT"

Reply:
renamed